src/fiatExchanges/utils.tsx

```diff
@@ -291,3 +294,78 @@ export const filterProvidersByPaymentMethod = (
 ) => {
   return externalProviders?.find((quote) => quote.paymentMethods.includes(paymentMethod))
 }
+
+export interface ProviderSelectionAnalyticsData {
+  centralizedExchangesAvailable: boolean
+  totalOptions: number
+  paymentMethodsAvailable: Record<PaymentMethod, boolean>
```

Comment:
Any particular reason for this structure as opposed to just PaymentMethod[] ?

Reply:
yeah, I think the SQL queries will be easier this way

Reply:
well, certain kinds of SQL queries anyway